runtime/src/main/java/org/corfudb/protocols/service/CorfuProtocolMessage.java

```diff
@@ -0,0 +1,139 @@
+package org.corfudb.protocols.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.proto.Common.UuidMsg;
+import org.corfudb.runtime.proto.ServerErrors.ServerErrorMsg;
+import org.corfudb.runtime.proto.service.CorfuMessage.PriorityLevel;
+import org.corfudb.runtime.proto.service.CorfuMessage.ProtocolVersion;
+import org.corfudb.runtime.proto.service.CorfuMessage.HeaderMsg;
+import org.corfudb.runtime.proto.service.CorfuMessage.RequestMsg;
+import org.corfudb.runtime.proto.service.CorfuMessage.RequestPayloadMsg;
+import org.corfudb.runtime.proto.service.CorfuMessage.ResponseMsg;
+import org.corfudb.runtime.proto.service.CorfuMessage.ResponsePayloadMsg;
+
+import java.util.UUID;
+
+import static org.corfudb.protocols.CorfuProtocolCommon.*;
```

Comment:
should we expand this? I think only the getUuidMsg is used

Reply:
Done